### DISCUSSION TITLE
Keycloak release lifecycle

### DISCUSSION THREAD

**Discussion by Doctor-love** [keycloak/keycloak#10449] [Q&A] [answered]
I'm curious about the lifecycle for Keycloak releases - more specifically if/how previous major versions are maintained.

Let's say that a new vulnerability like log4shell is discovered and that it affects all released Keycloak versions.
Will previous major versions (like 15 or 16) get point releases to address the issue and if so how many majors back?
Is there any documentation available related to the release lifecycle, such as https://ubuntu.com/about/release-cycle ?

My apologies if this information is already documented somewhere, at least I haven't been able to find it.

**Comment by stianst** [comment, marked as answer]
Please see https://www.keycloak.org/security.html. Only the latest major release will get an update.
